src/main/java/nl/tudelft/contextproject/Bomb.java

```diff
@@ -0,0 +1,63 @@
+package nl.tudelft.contextproject;
+
+import java.awt.Graphics2D;
+
+import com.jme3.material.Material;
+import com.jme3.math.ColorRGBA;
+import com.jme3.math.Vector3f;
+import com.jme3.scene.Geometry;
+import com.jme3.scene.Node;
+import com.jme3.scene.Spatial;
+import com.jme3.scene.shape.Sphere;
+
+/**
+ * Class representing the player wearing the VR headset.
+ */
+public class Bomb extends Entity {
+	private Geometry geometry;
+	private Spatial sp;
+	/**
+	 * Constructor for a default player.
+	 * This player is (for now) a red sphere.
+	 */
+	public Bomb() {
+		Node nod = (Node) Main.getInstance().getAssetManager().loadModel("Models/Bomb.j3o");
+		Material mat = new Material(Main.getInstance().getAssetManager(), "Common/MatDefs/Misc/Unshaded.j3md");
+		mat.setColor("Color", ColorRGBA.Red);
+		nod.setMaterial(mat);
+		Material matb = new Material(Main.getInstance().getAssetManager(), "Common/MatDefs/Misc/Unshaded.j3md");
+		matb.setColor("Color", ColorRGBA.White);
+		((Node) nod.getChild("Cylinder.001")).getChild(0).setMaterial(matb);
+		sp = nod;
+	}
+
+	@Override
+	public Geometry getGeometry() {
+		return geometry;
+	}
+	public Spatial getSpatial(){
+		return sp;
+	}
+
+	@Override
+	public void simpleUpdate(float tdf) {
+		sp.move(0, 0, 0);
+	}
+
+	@Override
+	public void mapDraw(Graphics2D g, int resolution) {
+		Vector3f trans = geometry.getLocalTranslation();
```

Comment:
@BCrlrd The code just happens to be the same at this point in time. VRPlayer and Bomb do different things. It is likely that VRPlayer's code for drawing and updating changes when controls are implemented, so I don't think that they should share the same base class with code.